pkg/kubelet/container/runtime.go

```diff
@@ -284,6 +284,24 @@ func (p *Pod) FindContainerByName(containerName string) *Container {
 	return nil
 }
 
+// ToAPIPod converts Pod to api.Pod. Note that if a field in api.Pod has no
+// corresponding field in Pod, the field would not be populated.
+func (p *Pod) ToAPIPod() *api.Pod {
+	var pod api.Pod
+	pod.UID = p.ID
```

Comment:
ugh, ideally we would keep api.Pod and kubelet/container.Pod close enough that pkg/conversion could do this automatically :/

Not asking for a change.